azure-resources/RecoveryServices/vaults/recommendations.yaml

```diff
@@ -81,3 +81,31 @@
       url: "https://learn.microsoft.com/azure/backup/backup-rbac-rs-vault#minimum-role-requirements-for-azure-vm-backup"
     - name: Recovery Services Vault
       url: "https://learn.microsoft.com/azure/backup/backup-azure-arm-vms-prepare"
+
+- description: Enable Soft Delete for Recovery Services Vaults in Azure Backup
+  aprlGuid: 9e39919b-78af-4a0b-b70f-c548dae97c25
+  recommendationTypeId: null
+  recommendationControl: Disaster Recovery
+  recommendationImpact: Medium
+  recommendationResourceType: Microsoft.RecoveryServices/vaults
+  recommendationMetadataState: Active
+  longDescription: |
```

Comment:
Please reword the longDescription to be 300 characters or less, including spaces.